src/macaron/config/global_config.py

```diff
@@ -79,6 +101,20 @@ def load_expectation_files(self, exp_path: str) -> None:
 
         self.expectation_paths = exp_files
 
+    def load_python_venv(self, venv_path: str) -> None:
+        """
+        Load Python virtual environment.
+
+        Parameters
+        ----------
+        venv_path : str
+            The path to the Python virtual environment of the target software component.
+        """
+        if os.path.isdir(venv_path):
+            logger.info("Successfully loaded %s", venv_path)
```

Comment:
I can't help but feel that checking that the path exits does not really count as a "load" operation.